node/pkg/websocketfetcher/providers/mexc/utils.go

```diff
@@ -23,9 +23,16 @@ func ResponseToFeedDataList(response BatchResponse, feedMap map[string]int32) ([
 		if err != nil {
 			return feedDataList, err
 		}
+
+		volume, err := common.VolumeStringToFloat64(item.QuoteVolume)
+		if err != nil {
+			return feedDataList, err
```

Comment:
@nick-bisonai This is just my personal preference for pattern matching when I am reading code. When there is an error and function is supposed to return some value, I try to return empty value.

```suggestion
			return nil, err
```

Reply:
I've been burnt by returning nil before few times since nil value and empty structure list value is checked differently from the test code, if you insist, I can update into nil, but I personally prefer this approach

Reply:
I am not sure how you got burnt, but maybe you should be because you should not have touched that value when it returns error :)

Reply:
I'll maybe refactor this later all together but not now